modules/terraform/azure/key-vault/outputs.tf

```
@@ -0,0 +1,20 @@
+output "key_vault_id" {
+  description = "Key Vault ID (used for RBAC role assignments)"
+  value       = try(azurerm_key_vault.kv[0].id, null)
+}
+
+output "key_ids" {
+  description = "Map of Key names to Key IDs: {key_name => key_id}"
+  value = {
+    for k, v in azurerm_key_vault_key.kms_key :
+    v.name => v.id
+  }
+}
+
+output "key_resource_ids" {
+  description = "Map of Key names to Key Resource IDs: {key_name => key_resource_id}"
+  value = {
+    for k, v in azurerm_key_vault_key.kms_key :
+    v.name => v.resource_id
+  }
+}
```

Combine both into 1 output

```suggestion
output "key_ids" {
  description = "Map of Key names to Key Value Object: {key_name => key_v alue_object}"
  value = {
    for k, v in azurerm_key_vault_key.kms_key :
   k => v
  }
}
```